.github/workflows/build.yml

```diff
@@ -117,15 +117,15 @@ jobs:
           rm -rf /usr/local/bin/2to3 
           
           brew update
-          brew install pkg-config pandoc boost ccache coreutils
+          brew install pkg-config pandoc boost ccache coreutils openlibm
           sudo xcode-select -switch /Applications/Xcode_${{ matrix.version }}.app
           ccache --set-config=cache_dir=$HOME/.ccache
 
     - name: Install (Windows meson cross compile)
       if: matrix.name == 'windows'
       run: |
           sudo apt-get update
-          sudo apt-get install -y pandoc libcairo2-dev ccache dos2unix zstd
```

Comment:
I would think that we don't need this for a cross-compile?

Reply:
I don't think there's an alternative package without them.

Reply:
oh wait nevermind, I see what you're saying.